### PR TITLE
Mta 1478 mta web reviewing apps

### PR DESCRIPTION
**Pull request by HagayVider1**
MTA-1478
Replaces PR: https://github.com/windup/windup-documentation/pull/769

### CODE REVIEW

File: docs/topics/mta-web-reviewing-apps.adoc [hunk 1]
```diff
@@ -11,16 +11,22 @@ You can use the {ProductName} ({ProductShortName}) {WebName} to determine the mi
...
 +
 [NOTE]
 ====
 Only one application can be reviewed at a time.
 ====
-. Click  *Review*.
-. Select *Proposed action* and *Effort estimate* from the lists. Set *Business criticality* and *Work priority*.
-. Click *Submit review*.
+
+. Click image:kebab.png[] (three dots) at the right end of the row, and select *Review* from the drop-down menu. The application *Review* parameters appear in the main pane.
```
Comment (anarnold97) [inline docs/topics/mta-web-reviewing-apps.adoc:21]: @sbeskin-redhat - please correct me if I am wrong here but is this not the `Options` menu

File: docs/topics/mta-web-reviewing-apps.adoc [hunk 2]
```diff
@@ -11,16 +11,22 @@ You can use the {ProductName} ({ProductShortName}) {WebName} to determine the mi
...
 +
+image::mta-web-assessment-review.png[Application review]
+
+. Click *Proposed action* and select the action.
+. Click *Effort estimate* and set the level of effort required to perform the assessment with the selected questionnaire.
+. In the *Business criticality* field enter how critical the application is to the business.
```
Comment (anarnold97) [inline docs/topics/mta-web-reviewing-apps.adoc:27]:
```suggestion

. In the *Business criticality* field, enter how critical the application is to the business.
```
Missing comma

File: docs/topics/mta-web-reviewing-apps.adoc [hunk 3]
```diff
@@ -11,16 +11,22 @@ You can use the {ProductName} ({ProductShortName}) {WebName} to determine the mi
...
+. In the *Work priority* field enter the application's priority.
```
Comment (anarnold97) [inline docs/topics/mta-web-reviewing-apps.adoc:28]:
```suggestion
. In the *Work priority* field, enter the application's priority.
```
missing comma

File: docs/topics/mta-web-reviewing-apps.adoc (hunk 1 above)
Comment (sbeskin-redhat) [reply docs/topics/mta-web-reviewing-apps.adoc:21]: @anarnold97 Yes, correct. It should be "the Options menu {kebab}".  
{kebab} is in the attributes - :kebab: image:kebab.png[title="Options menu"]

Comment (anarnold97) [inline docs/topics/mta-web-reviewing-apps.adoc:21]:
```suggestion
. Click image:kebab.png[Options menu] at the right end of the row, and select *Review* from the drop-down menu. The application *Review* parameters appear in the main pane.
```

File: docs/topics/mta-web-reviewing-apps.adoc [hunk 4]
```diff
@@ -11,16 +11,23 @@ You can use the {ProductName} ({ProductShortName}) {WebName} to determine the mi
...
-. Select *Proposed action* and *Effort estimate* from the lists. Set *Business criticality* and *Work priority*.
-. Click *Submit review*.
+
+. Click image:kebab.png[Options menu] at the right end of the row, and select *Review* from the drop-down menu. The application *Review* parameters appear in the main pane.
 +
+image::mta-web-assessment-review.png[Application review]
+
+. Click *Proposed action* and select the action.
+. Click *Effort estimate* and set the level of effort required to perform the assessment with the selected questionnaire.
+
```
Comment (mguetta1) [inline docs/topics/mta-web-reviewing-apps.adoc:27]: Hi @HagayVider1, looks like there is an redundant empty line here
Other than that, LGTM, thanks.

**Review by mguetta1 — APPROVED** [review]

File: docs/topics/images/mta-web-assessment-review.png
Comment (jmle) [inline docs/topics/images/mta-web-assessment-review.png:1]: Should this image contain the step numbers?

File: docs/topics/mta-web-reviewing-apps.adoc [hunk 5]
```diff
@@ -11,16 +11,22 @@ You can use the {ProductName} ({ProductShortName}) {WebName} to determine the mi
...
 +
 [NOTE]
 ====
 Only one application can be reviewed at a time.
 ====
-. Click  *Review*.
-. Select *Proposed action* and *Effort estimate* from the lists. Set *Business criticality* and *Work priority*.
-. Click *Submit review*.
+
+. Click image:kebab.png[] (three dots) at the right end of the row, and select *Review* from the drop-down menu. The application *Review* parameters appear in the main view.
```
Comment (anarnold97) [inline docs/topics/mta-web-reviewing-apps.adoc:21]: Wrong format

**Pull request closed without merging**